SOOP/SOOP_XBT/DELAYED/xbt_dm_imos_conversion.py

```diff
@@ -407,15 +407,17 @@ def raw_for_ed_path(netcdf_file_path):
 
 
 def create_filename_output(gatts, data):
+    filename = 'XBT_T_%s_%s_FV01_ID-%s' % (data['TIME'].strftime('%Y%m%dT%H%M%SZ'), gatts['XBT_line'], gatts['XBT_uniqueid'])
+    
+    #decide what prefix is required
     names = read_section_from_xbt_config('VARIOUS')
     str = names['FILENAME']
     if str == 'Cruise_ID':
         str = gatts['XBT_cruise_ID']
-
-    filename = '%s-XBT_T_%s_%s_FV01_ID-%s' % (str, data['TIME'].strftime('%Y%m%dT%H%M%SZ'), gatts['XBT_line'], gatts['XBT_uniqueid'])
-
-#    if data['TIME'] > datetime(2008, 0o1, 0o1):
```

Comment:
regarding this part, not that I particularly care and remember exactly why the filename was different after this arbitrary date (I would have to chat with Seb), but do you want in the future to reprocess the whole dataset?

Reply:
I think the name difference is because funding for IMOS was only after 2008? Hence the IMOS label on the later data and the XBT-SOOP on the earlier data.
I don't know if we need to keep the different names or not - need to check with Seb, I think.